src/stores/UserStore.ts

```diff
@@ -67,6 +71,23 @@ export const useUserStore = defineStore('UserStore', () => {
     return null;
   });
 
+  const accessToken = ref<string>();
+  const isAccessTokenExpired = computed<boolean>(() => {
+    if (accessToken.value == null) {
+      return true;
+    }
+    const exp = jwtDecode<JwtPayload>(accessToken.value)?.exp;
+    if (!exp) {
+      return true;
+    }
+
+    // get timestamp in seconds minus 1 minute to account for clock drift
+    const now = Math.floor(DateTime.now().minus({ minutes: 1 }).toSeconds());
+    const expiration = Math.floor(DateTime.fromSeconds(exp).toSeconds());
+
+    return now >= expiration;
+  });
```

Comment:
Possible property simplification?

```suggestion
const isAccessTokenExpired = computed<boolean>(() => {
const exp = accessToken.value ? jwtDecode<JwtPayload>(accessToken.value)?.exp : null;
return !exp || DateTime.now().toSeconds() >= exp - 60;
});
```

Reply:
Nice! It was too verbose before.
I also added a try/catch since the `jwtDecode` method will throw if the token is invalid